base/db/R/get.new.site.R

```diff
@@ -0,0 +1,53 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##' Get new site info using provided site information
```

Comment:
Can you explain why this function is needed? Using the database to generate new IDs isn't going to help with the goal of enabling runs of `do_conversions` when no database is present at all.

Reply:
This version of file handles only when con != NULL.
I've updated the code to handle db-less ID handling. Can you check it out once?